readthedocs/builds/_docker.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
This is a weird filename... I think we should just put it in a file called `constants` right?

Reply:
I used `constants_docker.py`, 👍🏼